ui/src/utils/statusStyles.ts

```diff
@@ -10,7 +10,7 @@ import {
 import { EventStatus, FunctionRunStatus } from "../store/generated";
 
 export default function statusStyles(
-  status: EventStatus | FunctionRunStatus | null
+  status: EventStatus | FunctionRunStatus | "Registered" | null
```

Comment:
Yeah - I dislike this too, maybe I'll move to a type instead of a string, but it's kind of a temp hack to get this status thing working for a function itself that doesn't return a status from the server.